src/Home.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import ImageUploader from './ImageUploader';
+import TextUploader from './TextUploader';
+import VideoUploader from './VideoUploader'; // Import the VideoUploader component
+import Navbar from './components/Navbar';
+import './Home.css';
+
+function Home(): JSX.Element {
+    const [isDialogOpen, setIsDialogOpen] = useState(false);
+    const [mediaType, setMediaType] = useState<string | null>(null);
+    const [fileNames, setFileNames] = useState<string[]>([]);
+
+    const openDialog = (event: React.MouseEvent<HTMLButtonElement>) => {
+        const type = event.currentTarget.dataset.type;
+
+        if (type) {
+            setMediaType(type);
+            setIsDialogOpen(true);
+            setFileNames([]);
+        }
+    };
+
+    const closeDialog = () => {
+        setMediaType(null);
+        setIsDialogOpen(false);
+    };
+
+    return (
+        <div>
+            <div id='root'>
+                <Navbar />
```

Comment:
We can move this component rendering to the `App.tsx` file since we will always want our pages to have a navbar